pkg/certificates/certificates.go

```diff
@@ -85,21 +97,49 @@ type Token struct {
 // Claim holds information about the circumstances under which an authorization
 // token was claimed.
 type Claim struct {
-	IP         string
+	Addr       string
 	Timestamp  int64
 	Identity   *provider.PeerIdentity
 	SignedCert *x509.Certificate
 }
 
+// Client implements pb.CertificateClient
+type Client struct {
+	client pb.CertificatesClient
+}
+
 // NewServer creates a new certificate signing grpc server
-func NewServer(log *zap.Logger) pb.CertificatesServer {
+func NewServer(log *zap.Logger, ca *provider.FullCertificateAuthority, authDB *AuthorizationDB) pb.CertificatesServer {
 	srv := CertificateSigner{
-		Log: log,
+		Log:    log,
+		CA:     ca,
+		AuthDB: authDB,
 	}
 
 	return &srv
 }
 
+// NewClient creates a new certificate signing grpc client
+func NewClient(ident *provider.FullIdentity, address string) (*Client, error) {
+	tc := transport.NewClient(ident)
+	conn, err := tc.DialAddress(context.Background(), address)
```

Comment:
should this context be passed in?

Reply:
yeah i totally spaced that the whole thing that inspired me to make sure wiki syncing was set up between github.com/storj/docs and github.com/storj/storj/wiki was to add to the code style guide that unless the case is exceptional, you should never create a new background context and always take one from somewhere else. if we need a context here, NewClient should take one as an argument.

Reply:
i think i was looking at another file though